docs/source/stonesoup.deleter.rst

```diff
@@ -18,3 +18,9 @@ Time Based
 .. automodule:: stonesoup.deleter.time
     :show-inheritance:
     :inherited-members:
+
+Multi
```

Comment:
Suggest Composite, or multiple method, something more descriptive so that the reader understands immediately